plutus-ledger-api/plutus-ledger-api.cabal

```diff
@@ -186,10 +186,11 @@ test-suite plutus-ledger-api-plugin-test
 
 -- This is a nightly test, so it is an executable instead of test-suite to avoid
 -- running this in CI.
-executable evaluation-test
+executable test-onchain-evaluation
```

Comment:
I changed the name of this (I'm not sure why now) but now I'm wondering if I shouldn't have done that: it claims to be a nightly test so maybe it's mentioned in a configuration file somewhere else.

Reply:
It used to be a nightly test, but since we changed some infra, I think it is not anymore. @zliu41 knows better.